### PR TITLE
Core/Player: Fixed Weapon Skillup on low level mobs

### PR DESCRIPTION
**Pull request by DevRival**
**Changes proposed:** Fixed Weapon Skillup on low level mobs

**Target branch(es):** 1.x

Tests performed: (Does it build, tested in-game, etc)
Tested in-game , works as intended.

**Known issues and TODO list:** No issue.

### CODE REVIEW

**Comment by ShinDarth** [comment]
why comment?

File: src/game/Entities/Player/Player.cpp [hunk 1]
```diff
@@ -6331,8 +6331,8 @@ void Player::UpdateCombatSkills(Unit* victim, WeaponAttackType attType, bool def
     uint8 plevel = getLevel();                              // if defense than victim == attacker
     uint8 greylevel = Trinity::XP::GetGrayLevel(plevel);
     uint8 moblevel = victim->getLevelForTarget(this);
-    if (moblevel < greylevel)
-        return;
+    /*if (moblevel < greylevel)
+        return;*/ // Patch 3.0.8 (2009-01-20): You can no longer skill up weapons on mobs that are immune to damage.
```
Comment (ShinDarth) [inline src/game/Entities/Player/Player.cpp]: why comment?

Comment (DevRival) [inline src/game/Entities/Player/Player.cpp]: What to know how and where I took) If you want I can remove?

Comment (ShinDarth) [inline src/game/Entities/Player/Player.cpp]: no, just leave it there for now

**Pull request merged**